source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -166,7 +166,14 @@ export abstract class SerializableObject {
     }
 
     setCustomProperty(name: string, value: any) {
-        this._rawProperties[name] = value;
+        let deleteProperty = (typeof value === "string" && Utils.isNullOrEmpty(value)) || value === undefined || value === null;
```

Comment:
nit nit: shouldDeleteProperty might be more intuituve of a name here for the condition variable

Reply:
Yes, that's a very small nit :) Hopefully you don't mind if I don't fix.